aztec/src/main/kotlin/org/wordpress/aztec/watchers/DeleteMediaElementWatcherAPI25AndHigher.kt

```diff
@@ -13,11 +13,11 @@ class DeleteMediaElementWatcherAPI25AndHigher(aztecText: AztecText) : TextWatche
     private var deletedSpans = ArrayList<AztecMediaSpan>()
 
     override fun beforeTextChanged(text: CharSequence, start: Int, count: Int, after: Int) {
-        if (aztecTextRef.get()?.isTextChangedListenerDisabled() ?: true) {
+        if (aztecTextRef.get()?.isTextChangedListenerDisabled() != false) {
```

Comment:
This changes the logic of the original if condition. Was this intentional? The equivalent to what is happening originally would be:

```
if (aztecTextRef.get() == null || aztecTextRef.get()?.isTextChangedListenerDisabled() != false) {
```

Reply:
I went bottom to the top so I've answered this already in another place 👍